drivers/bridge/bridge.go

```diff
@@ -300,7 +300,8 @@ func (n *bridgeNetwork) isolateNetwork(others []*bridgeNetwork, enable bool) err
 			}
 		}
 
-		if thisV6 != nil && otherV6 != nil && !types.CompareIPNet(thisV6, otherV6) {
+		// TODO: Enable when full ipv6 table infrastructure is available
+		if false && thisV6 != nil && otherV6 != nil && !types.CompareIPNet(thisV6, otherV6) {
```

Comment:
As discussed, should we even completely avoid adding ipv4/v6 addresses to the iptables filter rules and instead use @mrjana 's idea of using device based filters ?